lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -205,6 +206,14 @@ def __init_pulsar_app( self, pulsar_conf_path ):
             log.info("Loading Pulsar app configuration from %s" % pulsar_conf_path)
             with open(pulsar_conf_path, "r") as f:
                 conf.update(yaml.load(f) or {})
+        if "job_metrics_config_file" not in conf:
+            conf["job_metrics"] = self.app.job_metrics
```

Comment:
Ah, ok, that's because Pulsar supports both configuration options (xref https://github.com/galaxyproject/pulsar/blob/master/pulsar/core.py#L129 ). Works for me.